src/platform/linux/kmsgrab.cpp

```diff
@@ -614,7 +614,7 @@ namespace platf {
         for (auto &entry : fs::directory_iterator { card_dir }) {
           auto file = entry.path().filename();
 
-          auto filestring = file.generic_u8string();
+          auto filestring = file.generic_string();
```

Comment:
generic_u8string now returns `basic_string<char8_t>`. since char8_t is unsigned, this is now incompatible with `std::string_view = std::basic_string_view<char>`. Assuming this goes in with the UCRT64 change our locale is not exactly guaranteed to be utf-8 either, and the comparison we're immediately making is to an ANSI string so the u8-ness seems mostly irrelevant here. Therefore, making this change so compilation succeeds.

Reply:
Good thing is that we don't use `file.generic_string();` on Windows. Yet...